### PR TITLE
fix(docs): add reference to useWorkspaces

### PR DESCRIPTION
**Pull request by andrestone**
This seems to be a very popular use case and there's no mention on the main README.

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -158,7 +158,8 @@ Run `lerna --help` to see all available commands and options.
       "npmClientArgs": ["--no-package-lock"]
     }
   },
-  "packages": ["packages/*"]
+  "packages": ["packages/*"],
+  "useWorkspaces": false
```
Comment (evocateur) [inline README.md:162]: Why would you explicitly configure a value to `false`? It's confusing the issue. I agree `useWorkspaces` is poorly documented, but it needs more clarity than this.

**Review by evocateur — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (andrestone) [reply README.md:162]: I don't get how 'false' is more or less confusing than 'true', but whatever suits the project best works for me.

Comment (evocateur) [reply README.md:162]: I mean, `false` is the default. I think people understand how to set boolean properties in JSON files?

What I meant, more specifically, is a separate section that describes the changes between multiple files in the monorepo. It's not just a change to the `lerna.json`, after all, when using Yarn (or npm v7) workspaces.

**Comment by JamesHenry** [comment]
Hi @andrestone thanks a lot for contributing to lerna and its community. I'm going to close this one as it is no longer applicable to the file in question.

We will be building out more guidance around using workspaces in general soon.

Many thanks 🙏

**Pull request closed without merging**